_posts/2018-09-12-september-office-hours.md

```diff
@@ -0,0 +1,72 @@
+---
+title: September Office Hours
+description: Multi-user OpenFaaS Cloud, Istio Service Mesh, and Stateless Microservices
+
+date: 2018-09-11
+image: /images/september-office-hours/team.jpg
+categories:
+  - office-hours
+  - community
+author_staff_member: burton
+dark_background: true
+---
+
+The OpenFaaS community holds regular office hours for anyone to join in as well as specialized discussions with project members, and core contributors. During these discussions, we talk about what's new in the project, where we're headed, and any other updates individual members may have.
+
+All of the office hours meetings are recorded and available for later viewing on YouTube.
+
+<iframe width="700" height="394" src="https://www.youtube.com/embed/gkKpercV6FA?rel=0" frameborder="0" allow="autoplay; encrypted-media" allowfullscreen></iframe>
+
+> We also maintain an [official channel](https://www.youtube.com/channel/UCdKi97g5FmzvrmtIp9FyOVA) that contains demos from some of the meetings
+
+## Project Goals and Purpose
+OpenFaaS is focused on developers first, easy for operators (_not_ a headache), and the great community aspect
+
+* Developer first
```

Comment:
In the call I mentioned some additional work we did to support this like the `faas-cli up` - we can link to the blog post?